### ISSUE TITLE
Week 8 Quiz Part 1 Questions and Answer Key do not Match

### ISSUE THREAD

**Issue by Hongyi6328** [nus-cs2103-AY2223S1/forum#325]
Question 11 appears different in the original quiz and in the answer key.
![image](https://user-images.githubusercontent.com/58073006/193510613-4bc8785d-7dcd-4e9f-9ca3-49fb4c99823e.png)
![image](https://user-images.githubusercontent.com/58073006/193510634-98d5983e-5892-46a5-ad7e-25c68dc18df7.png)

**Comment by damithc** [comment]
@Hongyi6328 the questions in that section are ordered randomly. So, you could be comparing two different questions. Let me know if that's not the case.

**Comment by Hongyi6328** [comment]
Actually many questions got shuffled in the answer key.

**Comment by damithc** [comment]
> Actually many questions got shuffled in the answer key.

Yes, that's by design (that section is set to 'shuffle' mode). Every time you go to the submission page, the order of questions will be different. So, the question 11 in the answers page is not the same question shown as question 11 in the quiz submission page.
